dali/base/dafdesc.cpp

```diff
@@ -2934,30 +2937,33 @@ bool setReplicateDir(const char *dir,StringBuffer &out,bool isrep,const char *ba
     unsigned count = 0;
     unsigned i;
     for (i=0;d[i]&&dir[i]&&(d[i]==dir[i]);i++)
-        if (isPathSepChar(dir[i])) {
+    {
+        if (isPathSepChar(dir[i]))
+        {
             match = i;
             count++;
         }
+    }
     const char *r = repDir?repDir:queryBaseDirectory(grp_unknown, isrep ? 1 : 0,os);
-    if (d[i]==0) {
-        if ((dir[i]==0)||isPathSepChar(dir[i])) {
+    if (d[i]==0)
+    {
+        if (dir[i]==0)
+        {
+            out.append(r);
+            return true;
+        }
+        else if (isPathSepChar(dir[i])) // should mean that the last char of d was not a pathsepchar
+        {
             out.append(r).append(dir+i);
```

Comment:
does r definitely have no trailing /?  Otherwise this could lead to double //

Reply:
that may be an additional issue. I'll check and probably add a check for tailing '/' on r too.